modules/identity-provider-provisioning-user-lookup-mapping.adoc

```diff
@@ -1,35 +1,37 @@
 // Module included in the following assemblies:
 //
-// * orphaned
+// * authentication/identity_providers/configuring-basic-authentication-identity-provider.adoc
```

Comment:
The `lookup` method feature applies to all identity provider types, not only the `basic-authentication-identity-provider.adoc`. Actually in https://docs.openshift.com/container-platform/3.6/install_config/configuring_authentication.html#identity_providers_methods , this module is placed as general for all identity providers, not specific to any single identity provider.